test/bun.js/install/bunx.test.ts

```diff
@@ -0,0 +1,64 @@
+import { spawn } from "bun";
+import {
+  afterEach,
+  beforeEach,
+  expect,
+  it,
+} from "bun:test";
+import { bunExe } from "bunExe";
+import { bunEnv as env } from "bunEnv";
+import { realpathSync } from "fs";
+import { mkdtemp, rm } from "fs/promises";
+import { tmpdir } from "os";
+import { join } from "path";
+
+let x_dir;
+
+beforeEach(async () => {
+  x_dir = realpathSync(await mkdtemp(join(tmpdir(), "bun-install.test")));
```

Comment:
I don't think this directory ends up being used

`bunx` installs into `/tmp/*-bunx`

Reply:
not used per se, just want a blank directoy to prevent `bunx` from picking up anything inside to execute instead &minus; an unlikely event but better be safe than sorry :)